Source/WebKit/NetworkProcess/cache/NetworkCacheIOChannelGLib.cpp

```diff
@@ -104,7 +104,7 @@ void IOChannel::read(size_t offset, size_t size, Ref<WTF::WorkQueueBase>&& queue
                 }
             }
         }
-        queue->dispatch([protectedThis = WTFMove(protectedThis), completionHandler = WTFMove(completionHandler)] {
```

Comment:
ASan didn't complain here but since the logic is the same, I also patched this call site.